Util/CellUtils/joinCell.m

```diff
@@ -0,0 +1,75 @@
+function [newcell] = joinCell(c1, c2),
+    %function [newcell] = joinCell(c1,c2),
+    %
+    % join two column/vector cell objects if c1|c2 is 1xN or Nx1
+    %
+    % Inputs:
+    %
+    % c1 - first cell
+    % c2 - second cell
+    %
+    % Outputs:
+    %
+    % newcell - the join of c1+c2
+    %
+    % Example:
+    % [newcell] = joincell({'a','b'},{'c','d'})
+    % assert(isequal(newcell,{'a','b','c','d'}))
+    %
+    % author: hugo.oliveira@utas.edu.au
+    %
+
+    % Copyright (C) 2019, Australian Ocean Data Network (AODN) and Integrated
+    % Marine Observing System (IMOS).
+    %
+    % This program is free software: you can redistribute it and/or modify
+    % it under the terms of the GNU General Public License as published by
+    % the Free Software Foundation version 3 of the License.
+    %
+    % This program is distributed in the hope that it will be useful,
+    % but WITHOUT ANY WARRANTY; without even the implied warranty of
+    % MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    % GNU General Public License for more details.
+    %
+    % You should have received a copy of the GNU General Public License
+    % along with this program.
+    % If not, see <https://www.gnu.org/licenses/gpl-3.0.en.html>.
+    %
+
+    narginchk(2, 2);
+    c1s = squeeze(c1);
+    c2s = squeeze(c2);
+    c1size = size(c1s);
+    c2size = size(c2s);
+
+    c1_not_column = c1size(1) ~= 1;
+    c1_not_row = c1size(2) ~= 1;
+    c2_not_column = c2size(1) ~= 1;
+    c2_not_row = c2size(2) ~= 1;
+
+    invalid_c1 = c1_not_column && c1_not_row;
+    invalid_c2 = c2_not_column && c2_not_row;
+    invalid_input = invalid_c1 || invalid_c2;
+
+    if invalid_input,
+        error('Cell is not a vector.')
```

Comment:
Try to always end you line with ';'. Matlab has an embedded code checker where something not optimal will be underlined in orrange. Hovering it gives a hint on how to write better code. In this case the '=' sign should be underlined. In general try to pay attention to this feedback from the Matlab editor, this can be very useful sometimes.

Reply:
My matlab editor/linter complains about other stuff but not about the missing ; in error calls. I'm resolving some warnings anyway